vendor/terraform-config-inspect/VERSION

```diff
@@ -0,0 +1 @@
+0.0.20201102131242+git0c45ba392e51
```

Comment:
Nice

Reply:
Is this the timestamp of the commit or when you ran it?

Reply:
As you can tell from looking at it, November 02 is not the timestamp of when I ran it. This is a slightly modified version of the version string the `go` modules use for version tracking. I expect it is the timestamp of the commit, plus of course the partial git SHA of the commit.